internal/common/config/clientconfig.go

```diff
@@ -34,3 +62,31 @@ func (c DisasterConfig) GetClientDisasterConfig() ClientDisasterConfig {
 		),
 	}
 }
+
+// GetClientIIGOConfig gets ClientIIGOConfig
+func (c IIGOConfig) GetClientIIGOConfig() ClientIIGOConfig {
+	return ClientIIGOConfig{
```

Comment:
since the entire iigo config is exposed linter is telling you to just return `c`

Reply:
Can I do that?

Reply:
Screw it, I did that